azure-bundle/NOTICE

```diff
@@ -28,3 +28,291 @@ To find the details that apply to this artifact see the accompanying LICENSE fil
 A list of contributors may be found from CREDITS(-2.x) file, which is included
 in some artifacts (usually source distributions); but is always available
 from the source code management (SCM) system project uses.
+
+--------------------------------------------------------------------------------
+
+NOTICE for Group: io.netty  Name: netty-buffer  Version: 4.1.115.Final
```

Comment:
Minor: It would be better to just state that this is the NOTICE for Netty, not each artifact individually.

Reply:
I'm using the same formatting as for other dependencies. Do you want me to change that now or do we wait after the releases ?